### ISSUE TITLE
Login doesnt work in dev build, but works after uploading to internal testing

### ISSUE THREAD

**Issue by UmarBhatPlaystrom** [googlesamples/google-signin-unity#253]

**Comment by UmarBhatPlaystrom** [comment]
I opened issue for those who faced a similar issue. 

Actually here how it works afaik: When you build .apk it is considered as local dev build and all you need for your sign in to work is to have a particular keystore in the build, BUT when you build .aab (for release) it is automatically considered as release build and (maybe) is signed with another key, so that sign in won't work until you upload the build to internal testing.
We faced the issue where we built .aab for testing before uploading to store and we had our sign in not working at all.

In short: 
- You want to test locally? Enter keystore, build .apk
- You want to test from internal testing? Build .aab (obviously)
